doc/arkode/guide/source/Usage/SPRKStep_c_interface/User_callable.rst

```diff
@@ -0,0 +1,1229 @@
+
+   +-----------------------------------------+-------------------------------------------+----------+
+   | Optional input                          | Function name                             | Default  |
+   +=========================================+===========================================+==========+
+   | Direction of zero-crossings to monitor  | :c:func:`SPRKStepSetRootDirection()`      | both     |
+   +-----------------------------------------+-------------------------------------------+----------+
+   | Disable inactive root warnings          | :c:func:`SPRKStepSetNoInactiveRootWarn()` | enabled  |
+   +-----------------------------------------+-------------------------------------------+----------+
+
+
+
+.. c:function:: int SPRKStepSetRootDirection(void* arkode_mem, int* rootdir)
+
+   Specifies the direction of zero-crossings to be located and returned.
+
+   :param arkode_mem: pointer to the SPRKStep memory block.
+   :param rootdir: state array of length *nrtfn*, the number of root
+      functions :math:`g_i`  (the value of *nrtfn* was supplied in
+      the call to :c:func:`SPRKStepRootInit()`).  If ``rootdir[i] ==
+      0`` then crossing in either direction for :math:`g_i` should be
+      reported.  A value of +1 or -1 indicates that the solver
+      should report only zero-crossings where :math:`g_i` is
+      increasing or decreasing, respectively.
+
+   :return:
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the SPRKStep memory is ``NULL``
+      * *ARK_ILL_INPUT* if an argument has an illegal value
+
+   **Notes:**
+      The default behavior is to monitor for both zero-crossing directions.
+
+
+
+.. c:function:: int SPRKStepSetNoInactiveRootWarn(void* arkode_mem)
+
+   Disables issuing a warning if some root function appears
+   to be identically zero at the beginning of the integration.
+
+   :param arkode_mem: pointer to the SPRKStep memory block.
+
+   :return:
+      * *ARK_SUCCESS* if successful
+      * *ARK_MEM_NULL* if the SPRKStep memory is ``NULL``
+
+   **Notes:**
+      SPRKStep will not report the initial conditions as a
+      possible zero-crossing (assuming that one or more components
+      :math:`g_i` are zero at the initial time).  However, if it appears
+      that some :math:`g_i` is identically zero at the initial time
+      (i.e., :math:`g_i` is zero at the initial time *and* after the
+      first step), SPRKStep will issue a warning which can be disabled with
+      this optional input function.
+
+
+
+
+
+.. _ARKODE.Usage.SPRKStep.InterpolatedOutput:
+
+Interpolated output function
+--------------------------------
+
+An optional function :c:func:`SPRKStepGetDky()` is available to obtain
+additional values of solution-related quantities.  This function
+should only be called after a successful return from
+:c:func:`SPRKStepEvolve()`, as it provides interpolated values either of
+:math:`y` or of its derivatives (up to the 5th derivative)
```

Comment:
I don't recall how many of these there were but if it's only a few (hopefully) it might be easier to make these changes at the same time. If there are a bunch of them then we could open an issue with what needs to be fixed and make those changes in a followup PR.